README.md

```diff
@@ -16,6 +16,34 @@ For details about configuration and deployment of RBD and CephFS CSI plugins, se
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
+
+Here is the matrix that describes the features supported by the different
+versions of Ceph CSI driver
+
+|              |        Features             | CSI driver Version | Kubernetes Version   |
+|--------------|-----------------------------|--------------------|----------------------|
+| **CephFS**   | Snapshot                    |         1.0        |       >=1.13         |
+|              | Clone from VolumeSnapshot   |          -         |       >=1.13         |
+|              | Clone from VolumeSource     |          -         |       >=1.13         |
+|              | Resize                      |          -         |         1.14         |
+|              | RWX                         |       0.3, 1.0     |       >=1.13         |
+|              |                             |                    |                      |
+| **RBD**      | Snapshot                    |       0.3, 1.0     |       >=1.13         |
+|              | Clone from VolumeSnapshot   |       0.3, 1.0     |       >=1.13         |
+|              | Clone from VolumeSource     |          -         |       >=1.13         |
+|              | Resize                      |          -         |         1.14         |
+|              | RWX(only block mode)        |         1.0        |       >=1.13         |
+|              | RWO                         |       0.3, 1.0     |       >=1.13         |
+
+
+The below matrix describes the compatibility of Ceph CSI driver with the CSI spec version
+
+| CSI driver Version | CSI spec version |
+|--------------------|------------------|
+|        0.3         |     v1.0.0       |
```

Comment:
need to add diver name which helps the user to figure out which ceph-csi driver version support which CSI spec version